src/pages/en/guides/migrate-to-astro/from-pelican.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Migrating from Pelican
+description: Tips for migrating an existing Pelican project to Astro
+layout: ~/layouts/MigrationLayout.astro
+stub: true
+framework: Pelican
+---
+
+[Pelican](https://getpelican.com) is an open-source static site generator built on Python.
+
+## Key Similarities between Pelican and Astro
+
+Pelican and Astro share some similarities that will help you migrate your project:
+
+- Pelican and Astro are both static-site generators, ideally suited to content-focused sites like blogs.
+
+- Pelican and Astro both have built-in support for writing in Markdown, including frontmatter YAML properties for page metadata. However, Astro has very few reserved frontmatter properties compared to Pelican. Even though many of your existing Pelican frontmatter properties will not be "special" in Astro, this metadata can be accessed and used by Astro components. So, you can continue to use your existing Markdown files and frontmatter values.
+
+## Key Differences between Pelican and Astro
+
+When you rebuild your Pelican site in Astro, you will notice some important differences:
+
+- Pelican can process HTML (`.html`) files for page content. Astro can create pages from `.astro` files, which are written using a syntax that is a superset of HTML. You can write `.astro` files using exclusively valid HTML. But, `.html` files are not recognized by Astro.
```

Comment:
This is not entirely accurate. It’s totally OK to drop a `.html` file in `src/pages/` it’s just not very powerful because you don’t get any Astro magic like JSX, imports, etc. (I think Astro _might_ still bundle `<script>` tags? Been a while since I tested, but anyway, I digress!)

I think we can add a bullet for content formats given the Python-oriented `.rst` support in Pelican and tweak this point to focus on templating.

```suggestion
- Pelican supports writing content in Markdown and reStructured Text (`.rst`). Astro supports creating pages from Markdown and MDX files, but does not support reStructured Text.

- Pelican uses HTML files and Jinja syntax for templating. Astro uses `.astro` files, which are written using a syntax that is a superset of HTML. All valid HTML is valid `.astro` syntax.
```